submissions/kazeemErinfolami/htmlForms/signUp.html

```diff
@@ -18,43 +20,67 @@
-        Registering an account gives you access to sweet promo and events
-        available to only registered customers
-      </p>
-      <p class="para">Already got an account? <a href="#">Sign in</a></p>
-    </div>
+        <h1 class="reg">Account</h1>
+
+        <p class="para">
+          Registering an account gives you access to sweet promo and events
+          available to only registered customers
+        </p>
+        <p class="para">Already got an account? <a href="#">Sign in</a></p>
+      </div>
     </section>
     <div class="container">
       <form id="formValidation">
         <div class="form">
           <label for="firstName" id="firstNameL">First Name :</label>
-          <input class="forminput" type="text" name="firstName" id="firstName"/>
+          <input
+            class="forminput"
+            type="text"
+            name="firstName"
+            id="firstName"
+          />
           <p class="error"></p>
           <label for="lastName" id="lastNameL">Last Name :</label>
-          <input class="forminput" type="text" name="lastName" id="lastName"/>
+          <input class="forminput" type="text" name="lastName" id="lastName" />
           <p class="error"></p>
           <label for="otherName" id="otherNameL">Other Name :</label>
-          <input class="forminput" type="text" name="otherName" id="otherName"/>
+          <input
+            class="forminput"
+            type="text"
+            name="otherName"
+            id="otherName"
+          />
           <p class="error"></p>
           <label for="phoneNo" id="phoneNoL">Phone No :</label>
-          <input class="forminput" type="text" name="phoneNo" id="phoneNo"/>
+          <input class="forminput" type="text" name="phoneNo" id="phoneNo" />
           <p class="error"></p>
           <label for="email" id="emailL">Email Address :</label>
-          <input class="forminput" type="email" name="email" id="email"/>
+          <input class="forminput" type="email" name="email" id="email" />
           <p class="error"></p>
           <label for="password" id="passwordL">Password :</label>
-          <input class="forminput" type="password" name="password" id="password"/>
+          <input
+            class="forminput"
+            type="password"
+            name="password"
+            id="password"
+          />
           <p class="error"></p>
-          <label for="confirmPassword" id="confirmPasswordL">Confirm Password :</label>
-          <input class="forminput" type="password" name="confirmPassword" id="confirmPassword"/>
+          <label for="confirmPassword" id="confirmPasswordL"
+            >Confirm Password :</label
+          >
+          <input
+            class="forminput"
+            type="password"
+            name="confirmPassword"
+            id="confirmPassword"
+          />
           <p class="error"></p>
         </div>
         <div class="agreeContainer">
           <label for="agree" id="agreeL" class="agree">
```

Comment:
format this label and input tag correctly. The input field I believe is nested inside the label tag